solarpark/persistence/__init__.py

```diff
@@ -209,3 +208,18 @@ def delete_all_member_data(db: Session, member_id: int):
         )
         create_error(db_error, error_request)
         return False
+
+    try:
+        last_item = db.query(Member).order_by(Member.id.desc()).first()
+        alter_sequence_query = f"ALTER SEQUENCE members_id_seq RESTART WITH {last_item.id+1}"
+        db.execute(text(alter_sequence_query))
+        db.commit()
+    except Exception as ex:
+        error_request = ErrorLogCreateRequest(
+            member_id=member_id,
+            comment=f"Error resetting member sequence after deletion of share {member_id}, details: {ex}",
```

Comment:
Borde det vara `f"Error resetting member sequence after deletion of member {member_id}, details: {ex}"`